packages/atomic/cypress/integration/search-box/instant-results/search-box-instant-results.cypress.ts

```diff
@@ -0,0 +1,186 @@
+import {TestFixture} from '../../../fixtures/test-fixture';
+import {
+  SafeStorage,
+  StorageItems,
+} from '../../../../src/utils/local-storage-utils';
+import {SearchBoxSelectors} from '../search-box-selectors';
+import {addSearchBox} from '../search-box-actions';
+import * as CommonAssertions from '../../common-assertions';
+import * as SearchBoxAssertions from '../search-box-assertions';
+import * as InstantResultsAssertions from './search-box-instant-results-assertions';
+// import {Result} from '@coveo/headless';
+import {InstantResultsSelectors} from './search-box-instant-results-selectors';
+
+// type PartialResultList = Partial<Result>[];
+const setInstantResults = (count: number) => () => {
+  cy.intercept({method: 'POST', path: '**/rest/search/v2?*'}, (request) => {
+    request.reply((response) => {
+      const newResponse = response.body;
+      newResponse.results = Array.from({length: count}, (_, i) => ({
+        title: `Instant Result ${i}`,
+        uniqueId: `instant_result_${i}`,
+        uri: `about:blank?${i}`,
+        clickUri: `about:blank?${i}`,
+        raw: {
+          urihash: `${i}`,
+        },
+      }));
+
+      response.send(200, newResponse);
+    });
+  }).as(TestFixture.interceptAliases.Search.substring(1));
+};
```

Comment:
This could be simplified like so:
```suggestion
const setInstantResults = (count: number) =>
  (fixture: TestFixture) => {
    fixture.withCustomResponse(response => {
      response.results = Array.from({length: count}, (_, i) => ({
        title: `Instant Result ${i}`,
        uniqueId: `instant_result_${i}`,
        uri: `about:blank?${i}`,
        clickUri: `about:blank?${i}`,
        raw: {
          urihash: `${i}`,
        },
      }));
    });
  };
```